internal/backends/sqlite/database.go

```diff
@@ -180,23 +180,35 @@ func relationStats(ctx context.Context, db *fsql.DB, list []*metadata.Collection
 		return nil, lazyerrors.Error(err)
 	}
 
-	placeholders := make([]string, len(list))
-	args := make([]any, len(list))
+	tablesPlaceholders := make([]string, len(list))
+	tablesArgs := make([]any, len(list))
+
+	indexesPlaceholders := make([]string, 0, len(list))
+	indexesArgs := make([]any, 0, len(list))
+
+	var indexes int64
 
 	for i, c := range list {
-		placeholders[i] = "?"
-		args[i] = c.TableName
+		tablesPlaceholders[i] = "?"
+		tablesArgs[i] = c.TableName
+
+		indexes += int64(len(c.Settings.Indexes))
+
+		for _, index := range c.Settings.Indexes {
+			indexesPlaceholders = append(indexesPlaceholders, "?")
+			indexesArgs = append(indexesArgs, c.TableName+"_"+index.Name)
+		}
 	}
 
 	q = fmt.Sprintf(`
 		SELECT SUM(pgsize) AS SizeTables
```

Comment:
```suggestion
		SELECT SUM(pgsize)
```

It is also unused now

Reply:
Then it won't be consistent with the changes in this PR https://github.com/FerretDB/FerretDB/pull/3363/files - `SizeTables` and `CountCells` don't need to be named either. Should I remove all the named params or only in the current PR?

Reply:
All of them. It looks like names are important, but they are not – we use the order of the columns, not names